### PR TITLE
Fonts: Add additional font steps

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adds additional font steps that will be used in Fabric 7's Fluent theme. This is to insure that Fabric 7's Fluent theme is backwards compatible with components coded against Fabric 6's theme API.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9329)

### CODE REVIEW

File: packages/styling/src/interfaces/IFontStyles.ts [hunk 1]
```diff
@@ -13,7 +13,17 @@ export interface IFontStyles {
   mediumPlus: IRawStyle;
   large: IRawStyle;
   xLarge: IRawStyle;
+  /**
+   * @deprecated Exists for forward compatibility with Fabric 7's Fluent theme.
```
Comment (phkuo) [inline packages/styling/src/interfaces/IFontStyles.ts]: what's the backwards version of deprecated?

Comment (Vitalius1) [reply packages/styling/src/interfaces/IFontStyles.ts]: @phkuo I think @detacerped would do the trick 😄

**Comment by Vitalius1** [comment]
Need a snapshot update to pass

File: packages/styling/src/interfaces/IFontStyles.ts (hunk 1 above)
Comment (KevinTCoughlin) [reply packages/styling/src/interfaces/IFontStyles.ts]: > @phkuo I think @detacerped would do the trick 😄

lol

**Comment by phkuo** [comment]
> Need a snapshot update to pass

Hmm, I'm running `npm run update-snapshots` but it doesn't actually any snapshots. Any ideas?

**Comment by natalieethell** [comment]
@phkuo it looks like the @uifabric/styling snapshots need to be updated, so maybe try running `npm run update-snapshots` in `packages/styling`

**Comment by phkuo** [comment]
@msft-github-bot auto-merge when ready

**Comment by phkuo** [comment]
@dzearing need your signoff

**Pull request merged**